utils/statd/hostname.c

```diff
@@ -200,6 +200,18 @@ statd_canonical_name(const char *hostname)
 	ai = get_addrinfo(hostname, &hint);
 	if (ai != NULL) {
 		/* @hostname was a presentation address */
+#if 1
+		/*
+		 * Use the presentation address for the file name.
```

Comment:
Could there be a case where one host is using multiple different IP addresses that we'd want to match to the same record?

Reply:
In the original code, an nfs client host that has multiple different IP addresses for NFS would map to the same record only if there were reverse lookup entries for those ip addresses.  The reverse lookup entries are not required and statd supports not having them. If they are not present then the notify list can have multiple entries to the same host.  Which only means that the host could get multiple notifications when the nfs server is restarted and locks were taken from different ip addresses.